test/native/protobufBufferTest.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "protobuf.h"
+#include "testRunner.hpp"
+#include <cstdint>
```

Comment:
What for?

Reply:
Do you have something which tells you that some headers are not needed?

3792066c4a29b3f9ed70655bad323d3426b3786c

Reply:
Intuition :) Also, a rule of thumb: if it is a C++ header, it is likely not needed :)